backend/src/main/java/gov/cdc/usds/simplereport/api/reporting/ConditionController.java

```diff
@@ -0,0 +1,25 @@
+package gov.cdc.usds.simplereport.api.reporting;
+
+import gov.cdc.usds.simplereport.service.ConditionService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class ConditionController {
+  private final ConditionService conditionService;
+
+  @GetMapping("/universal/conditions/sync")
+  public String syncConditions() {
+    conditionService.syncConditions();
+    return "Condition sync has been started successfully.";
```

Comment:
```suggestion
    try {
        conditionService.syncConditions();
    } catch (AccessDeniedException e) {
        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(null);
    }
    return "Condition sync has been started successfully.";
```

or something like that

Reply:
Any particular reason for explicitly returning "Access denied" instead of just the 401 Unauthorized response if the user doesn't have the global admin authorization? Worth noting that when running the backend locally, the `no-security` spring profile is active which will permit you to run these syncs without providing a local access token

Reply:
Nope, I just saw the method was returning a String and so I guess I was trying to have minimal impact, which probably makes more sense on production code and not a prototype 😆 

As for running locally, yes no token needed, but I did run into this exception until I set `simple-report.demo-users.site-admin-emails: bob@example.com` in my config